### PR TITLE
docs: touch up query API documentation

### PR DESCRIPTION
**Pull request by jsstevenson**
I don't know if this is, like, essential work, but I was thinking about it the other day and wanted to see how code blocks in docstrings looked in the sphinx-generated pages, so... here it is.

Also made some of the query methods actually private to make the API page cleaner. I am pretty sure most of this stuff is not intended to be used by external methods, like, ever but we could always refactor or make some of it more usable by other objects.

### CODE REVIEW

File: gene/query.py [hunk 1]
```diff
@@ -530,12 +534,23 @@ def _prepare_normalized_response(self, query: str) -> Dict[str, Any]:
...
+        Use to retrieve normalized gene concept records:
+
+        >>> from gene.query import QueryHandler
+        >>> from gene.database import create_db
+        >>> q = QueryHandler(create_db())
+        >>> result = q.normalize("BRAF")
+        >>> result.gene_descriptor.gene_id
+        'hgnc:1097'
+        >>> result.xrefs
+        ['ensembl:ENSG00000157764', 'ensembl:ENSG00000157764', 'ncbigene:673']
```
Comment (jsstevenson) [inline gene/query.py:546]: so, this is a bug

Comment (jsstevenson) [reply gene/query.py]: fixed!

**Comment by jsstevenson** [comment]
honestly the docs look pretty clean imo!
Screenshot 2023-05-01 at 3 09 47 PM

**Review by korikuzma — APPROVED** [review]

**Comment by korikuzma** [comment]
> honestly the docs look pretty clean imo! Screenshot 2023-05-01 at 3 09 47 PM

So clean 💯  Great work!

**Pull request merged**